hivemind/p2p/p2p_daemon.py

```diff
@@ -169,6 +170,12 @@ async def create(
 
         return self
 
+    async def add_unary_handler(self, handle_name: str, handler: p2pclient.TUnaryHandler):
+        return await self._client.add_unary_handler(handle_name, handler)
+
+    async def call_unary_handler(self, peer_id: PeerID, handle_name: str, data: bytes) -> bytes:
+        return await self._client.call_unary_handler(peer_id, handle_name, data)
+
```

Comment:
I don't think we need these public bytes-level methods since it's unlikely that a user may want to call them directly (so they their presence will only confuse the user).

The private versions also seem useless since other private methods can call `self._client.add/call_unary_handler()` directly.

```suggestion
```